pkg/chartutil/chartfile_test.go

```diff
@@ -92,12 +92,12 @@ func verifyChartfile(t *testing.T, f *chart.Metadata, name string) {
 		t.Fatalf("Unexpected annotations")
 	}
 
-	if value := f.Annotations["extrakey"]; value != "extravalue" {
-		t.Errorf("Expected extravalue, but got %s", value)
+	if want, got := "extravalue", f.Annotations["extrakey"]; want != got {
+		t.Errorf("Want %q, but got %q", want, got)
```

Comment:
- [ ] The Go style I'm used to does not capitalize these terse failure messages that are not complete sentences, but if the rest of the Helm project _does_ capitalize them, then this is fine as is.  
  
  I see that elsewhere in this file, the messages are capitalized.